generators/server/templates/src/main/resources/config/couchmove/changelog/V0.1__initial_setup/user__admin.json.ejs

```diff
@@ -3,11 +3,11 @@
 <%_ if (!authenticationTypeOauth2) { _%>
   "password": "$2a$10$gSAhZrxMllrbgj/kkK9UceBPpChGWJA7SYIb1Mqo.n5aNLq1/oRrC",
 <% } %>
-  "first_name": "Administrator",
-  "last_name": "Administrator",
+  "firstName": "Administrator",
+  "lastName": "Administrator",
   "email": "admin@localhost",
   "activated": true,
-  "lang_key": "<%= nativeLanguage %>",
+  "langKey": "<%= nativeLanguage %>",
   "created_by": "system",
```

Comment:
```suggestion
  "langKey": "<%= nativeLanguage %>",
  "created_by": "system",
```
Having fields camelCase and others snakeCake looks ugly to me.

Reply:
There was a bug in Spring Couchbase's `@Field` annotation that prevented specifying custom field keys for Couchbase, so the variable names had to be used instead. It seems like it may have been fixed very recently since @tchlyah used it in the entity relationships fix, so I can explore that and if I'm still running into issues I'll create an issue to explore that after this PR is merged.

Reply:
Fixed